test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -365,6 +366,76 @@ def get_supported_operators(cls) -> List[OperatorConfig]:
             m, expected_node_list=node_list, expected_node_occurrence=node_occurrence
         )
 
+    def test_fixed_qparams_qspec(self):
+        class M(torch.nn.Module):
+            def forward(self, x):
+                return torch.sigmoid(x)
+
+        class BackendAQuantizer(Quantizer):
+            def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
+                for node in model.graph.nodes:
+                    if (
+                        node.op == "call_function"
+                        and node.target == torch.ops.aten.sigmoid.default
+                    ):
+                        input_act = node.args[0]
+                        assert isinstance(input_act, Node)
+                        act_qspec = FixedQParamsQuantizationSpec(
+                            dtype=torch.uint8,
+                            quant_min=0,
+                            quant_max=255,
+                            qscheme=torch.per_tensor_affine,
+                            scale=2.0 / 256.0,
```

Comment:
Hi @jerryzh168, May I ask a question about this fixed scale and zp value of `sigmoid`? The output range of `sigmoid` is among 0-1, so why we set the fixed scale as `2.0 / 256.0` and fixed zp as `128`.

Reply:
oh sorry I think this is not correct, I was trying to copy from https://github.com/pytorch/pytorch/blob/main/torch/ao/quantization/backend_config/_common_operator_config_utils.py but was writing down the wrong parameters, thanks for the comment!